desktop-tool/src/constants.py

```diff
@@ -172,7 +172,17 @@ class TargetSites(Enum):
             Cardstocks.P10: "Kunststoff",
         },
     )
-
+    PrinterStudioFR = TargetSite(
+        base_url="https://www.printerstudio.fr",
+        starting_url_route="personnalise/cartes-de-jeu-sur-mesure-cartes-blanches.html",
+        supports_foil=False,
+        cardstock_site_name_mapping={
+            Cardstocks.S30: "Standard (lisse)",
+            Cardstocks.S33: "Superieur (lisse)",
```

Comment:
does this work with these accents?
<img width="236" alt="image" src="https://github.com/user-attachments/assets/783b81a3-89ea-4765-a673-dc876d7898a6" />